test/shapes/test_star.py

```diff
@@ -0,0 +1,101 @@
+from pathlib import Path
+
+import fpdf
+from test.conftest import assert_pdf_equal
+
+import pytest
+
+
+HERE = Path(__file__).resolve().parent
+
+
+def test_star(tmp_path):
```

Comment:
The tests are great!

When arguments passed to a method are "hardcoded" values and not named variables,
I find it clearer to pass options by name:
```python
df.star(x=15, y=y, r_in=5, r_out=15, rotateDegrees=3, style="D")
```
Making this change here is entirely up to you, I'm OK to merge this file as it is.

Reply:
Yes, I agree with you, I will keep in mind also for next times